docs/release_guide.md

```diff
@@ -47,8 +47,23 @@ $ git push upstream release-<version> # ex: git push upstream release-v0.4
 ## Add changes to be backported
 
 Create a new branch off of the release branch to maintain updates specific to the new version. Let's call it the patch branch. The patch branch should not be created in the upstream repo.
-
+```console
+$ git checkout upstream/release-<version> # ex: git checkout upstream/release-v1.2
+$ git switch -c <patch-branch-name> # Create the patch branch in your repo 
+```
 If there are other commits on the `main` branch to be included in the release (such as for successive release candidates or patch releases), cherry-pick those onto the patch branch.
+```console
+$ git cherry-pick <commit-id> # ex: git cherry-pick 1bdc75a
+```
+Fix any merge conflicts and change the commit message to begin with [backport]:
+```bash
+[backport] allow exporting access log to OTEL collector # previously - telemetry: allow exporting access log to OTEL collector
+```
+Push your changes to your branch.
+```console
+$ git push <local-repo> <patch-branch-name>
+```
+ Create a pull request to backport the changes to the release branch, see [example PR](https://github.com/openservicemesh/osm/pull/5210).
```

Comment:
This is so helpful!! Thank you! Could you also add a comment about using a merge commit instead of a squash when merging to preserve the original commits?